app/public/lobby/lobbyfuncs.js

```diff
@@ -68,21 +99,69 @@ refreshBtn.addEventListener("click", () => {
 			console.log("ERROR - Something went wrong");
 		}
 		// update table according to list
+		resp.json().then((lobbyList) => {
+			console.log(lobbyList);
+			refreshLobbies(lobbyList);
+		});
+		
 	}).catch((e) => {
 		console.log(e);
 	});
```

Comment:
This code is identical to the code above. I recommend making a `fetchRefresh()` function that is called both by the refresh button and when connecting to the websocket